README.md

```diff
@@ -1,128 +1,207 @@
 - Chinese translations: [CrestedPeak9](https://twitter.com/CrestedPeak9), maksim71_doll, DeepL
 - Japanese translations: [Yu-miya](https://www.twitch.tv/toho_yumiya), CyrusVorazan, DeepL
-- Transferring all reports from the [old bugtracker](https://github.com/ack7139/thprac/issues) to [here](https://github.com/touhouworldcup/thprac/issues): [toimine](https://www.youtube.com/channel/UCtxu8Rg0Vh8mX6iENenetuA)
-- Chinese README translations: [TNT569](https://github.com/TNT569) 
+- Moving all reports from the [old bugtracker](https://github.com/ack7139/thprac/issues) [here](https://github.com/touhouworldcup/thprac/issues): [toimine](https://www.youtube.com/channel/UCtxu8Rg0Vh8mX6iENenetuA)
+- Chinese README.md translations: [TNT569](https://github.com/TNT569) 
+
+## <a name="faq"></a> **Frequently Asked Questions (FAQ)**
+
+## General
+
+### Why did thprac move?
+The original developer, Ack, has indefinitely ceased all future development of thprac/Marketeer. Below is their statement:
+> I will indefinitely cease all future development on thprac/Marketeer, you are free to continue working on it, granting that you follow the license's terms.  
+> Due to my insufficient skill, the code is glutted with obscure writings and terrible logic, turning the whole thing into a complete mess. I'm sorry if that disturbs you.  
+
+Ack cannot be contacted at this time, and development has been taken over by others.
+
+---
+
+### My antivirus says thprac is malware. Is it safe?
+Thprac contains no malicious code, but its behavior may trigger your antivirus. If flagged, add an exception or trust rule in your antivirus to allow thcrap to work. Alternatively, you can try [this version](https://github.com/touhouworldcup/thprac/issues/112) and report back if the issue persists.
+
+---
+
+## Compatibility
+
+### Does thprac work with English patches?
+Thprac is compatible with **thcrap** and can function as a thcrap launcher, similar to the [Universal THCRAP Launcher](https://github.com/thpatch/Universal-THCRAP-Launcher/). It does not support static English patches, like those created by gensokyo.org.
+
+---
+
+### thprac cannot find Embodiment of Scarlet Devil. What should I do?
+Ensure your executable is named either `東方紅魔郷.exe` or `th06.exe`. If thprac still fails to detect it, you can manually launch the game with any filename through the launcher, which will attach thprac to the game.
+
+---
+
+## Features
+
+### How do I save a replay mid-game in TH06-10?
+Saving replays mid-game is not supported due to technical limitations. As per the thprac 2.0.8.3 manual:
+> Because of how these games were programmed, adding this feature would be quite troublesome, so currently there's no direct solution to this.  
+
+However, EoSD now supports this functionality, albeit with quirks. 
+
+Note: replays saved mid-game do not return to the title screen after the save point.
```

Comment:
The note "albeit with quirks" is not really needed when the one and only quirk is described right below